k8s/perftest/common/ia/case-notifications-api.yaml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: case-notifications-api
+  namespace: ia
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: ia-case-notifications-api
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: ia-case-notifications-api
+    version: 0.0.2
+  values:
+    java:
+      replicas: 2
+      memoryRequests: '768Mi'
+      cpuRequests: '500m'
+      memoryLimits: '2048Mi'
+      cpuLimits: '1000m'
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "4315969c-54e6-43eb-b1c7-2be6fd411b82"
+      image: hmctspublic.azurecr.io/ia/case-notifications-api:prod-8c529b3f
+      ingressHost: ia-case-notifications-api-perftest.service.core-compute-perftest.internal
```

Comment:
this  can be set in your charts values.yaml as:
```
ia-case-notifications-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
```

saves having to update it in every env